hawk.js

```diff
@@ -2319,14 +5423,143 @@ Handlebars.registerPartial("NestedCheckbox", NESTED_CHECKBOX_TEMPLATE);
+  var hawk_banner = document.getElementById('hawk-top-banner').innerHTML = _HawkTopBanner_();
+
+  
+  //top pagination
+  const Hawk_Top_Pagination = `<div id="hawktoppager">
+                                <div class="hawk-listingControls">
+                                  <div class="hawk-mainControls clearfix">
+                                      <div id="hawk-pagination-top-bar"></div>
+                                  </div>
+                                  <div class="hawk-subControls clearfix">
+                                      <div id="hawk-compare-items-bar"></div>
+                                  </div>
+                                </div>
+                              </div>`;
+  var _HawkTopPagination_ = Handlebars.compile(Hawk_Top_Pagination);
+  var hawkTopPagination = document.getElementById('hawk-top-pagination').innerHTML = _HawkTopPagination_();
+  //end top pagination
+
+  //start result
+  const Hawk_Results = `<div id="hawk-result-items-container" class="hawk-results itemList productList clearfix s_1half sw_1third m_1third ui-sortable"></div>`;
+  var _HawkResult_ = Handlebars.compile(Hawk_Results);
+  var hawkResult = document.getElementById('hawk-result-items').innerHTML = _HawkResult_();
+  //end Result
+
+  //bottom Pagination
+  const Hawk_Bottom_Paginaton = `<div class="hawk-listingControls">
+                                    <div class="hawk-mainControls clearfix">
+                                        <div id="hawk-pagination-bottom-bar"></div>
+                                    </div>
+                                  </div>`;
+  var _HawkBottomPagination_ = Handlebars.compile(Hawk_Bottom_Paginaton);
+  var hawkBottomPagination = document.getElementById('hawk-bottom-pagination').innerHTML = _HawkBottomPagination_();
+  //end bottom Pagination
+
+  //start Hawk Content Display
+  const DISPLAY_CONTENT = `
+      <div id="hawk-auto-correct-keyword-list" class="hawk-auto-correct-keyword-container"></div>
+      <div id="hawk-top-banner"></div>
+      <div id="hawk-tabs-bar"></div>
+      <div id="hawktoppager">
+          <div class="hawk-listingControls">
+          <div class="hawk-mainControls clearfix">
+              <div id="hawk-pagination-top-bar"></div>
+          </div>
+          <div class="hawk-subControls clearfix">
+              <div id="hawk-compare-items-bar"></div>
+          </div>
+          </div>
+      </div>
+      <div id="hawk-bottom-banner"></div>
+      <div id="hawk-bottom2-banner"></div>
+      <div id="hawk-result-items-container"
+          class="hawk-results itemList productList clearfix s_1half sw_1third m_1third ui-sortable"></div>
+      <!-- <div id="hawk-pagination-bottom-bar"></div> -->
+      <div>
+          <div class="hawk-listingControls">
+          <div class="hawk-mainControls clearfix">
+              <div id="hawk-pagination-bottom-bar"></div>
+          </div>
+          </div>
+      </div>
```

Comment:
Move all these templates back to html. User will need to read the whole file if he wants to remove the bottom pagination bar.